<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshingSetupDialog</class>
 <widget class="QDialog" name="MeshingSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meshing options</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_34">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_34">
    <item>
     <widget class="QLineEdit" name="leGrading">
      <property name="text">
       <string>0.3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_53">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_53">
    <item>
     <widget class="QLabel" name="laEdgeAngle">
      <property name="text">
       <string>1.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_35">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_35">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Elements per curvature radius:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_51">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_51">
    <item>
     <widget class="QSlider" name="hsSurfaceCurvature">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_55">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_55">
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>STL - edge angle:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_37">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_37">
    <item>
     <widget class="QSlider" name="hsEdge">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_42">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_42">
    <item>
     <widget class="QSlider" name="hsChartDist">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_44">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_44">
    <item>
     <widget class="QLabel" name="laLineLength">
      <property name="text">
       <string>0.6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_32">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_32">
    <item>
     <widget class="QLineEdit" name="leMax">
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_56">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_56">
    <item>
     <widget class="QLabel" name="laSurfaceMeshCurv">
      <property name="text">
       <string>2.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_14">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLineEdit" name="leMin">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_48">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_48">
    <item>
     <widget class="QSlider" name="hsCloseEdges">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_45">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_45">
    <item>
     <widget class="QSlider" name="hsLineLength">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_40">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_40">
    <item>
     <widget class="QLabel" name="laEdge">
      <property name="text">
       <string>1.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_38">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_38">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Elements per edge:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_39">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_39">
    <item>
     <widget class="QLabel" name="laRadius">
      <property name="text">
       <string>2.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_46">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_46">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>STL - line length:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_50">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_50">
    <item>
     <widget class="QLabel" name="laSurfaceCurvature">
      <property name="text">
       <string>1.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_43">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_43">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>STL - chart distance:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_54">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_54">
    <item>
     <widget class="QSlider" name="hsEdgeAngle">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_58">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_58">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>STL - surface mesh curv:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_31">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_31">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Max mesh-size:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min mesh-size:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_57">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_57">
    <item>
     <widget class="QSlider" name="hsSurfaceMeshCurv">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_47">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_47">
    <item>
     <widget class="QLabel" name="laCloseEdges">
      <property name="text">
       <string>2.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_49">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_49">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>STL - close edges:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_41">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_41">
    <item>
     <widget class="QLabel" name="laChartDistance">
      <property name="text">
       <string>1.6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_52">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_52">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>STL - surface curvature:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_36">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_36">
    <item>
     <widget class="QSlider" name="hsRadius">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TickPosition::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_33">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_33">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Mesh-size grading:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leMin</tabstop>
  <tabstop>leMax</tabstop>
  <tabstop>leGrading</tabstop>
  <tabstop>hsRadius</tabstop>
  <tabstop>hsEdge</tabstop>
  <tabstop>hsChartDist</tabstop>
  <tabstop>hsLineLength</tabstop>
  <tabstop>hsCloseEdges</tabstop>
  <tabstop>hsSurfaceCurvature</tabstop>
  <tabstop>hsEdgeAngle</tabstop>
  <tabstop>hsSurfaceMeshCurv</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MeshingSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeshingSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
